### PR TITLE
Apply some clang-tidy fixes

### PR DESCRIPTION
**Pull request by bejado**

### CODE REVIEW

File: libs/utils/include/utils/Invocable.h [hunk 1]
```diff
@@ -18,6 +18,7 @@
 #define TNT_UTILS_INVOKABLE_H
 
 #include <type_traits>
+#include <utility>
```
Comment (romainguy) [inline libs/utils/include/utils/Invocable.h:21]: @pixelflinger Is that the type of STL includes we should strive to remove from our public headers?

**Review by romainguy — APPROVED** [review]

File: libs/utils/include/utils/Invocable.h (hunk 1 above)
Comment (pixelflinger) [reply libs/utils/include/utils/Invocable.h:21]: yeah, I'm not sure why this one is needed. We need to see if we can replace whatever it does easily. I think generally <utility> is okay though.

**Review by pixelflinger — COMMENTED** [review]
Is there a way we can avoid <utility> ?

**Comment by bejado** [comment]
`<utility>` is needed for `std::forward`

**Pull request merged**